src/NFe/Entity/Total.php

```diff
@@ -400,6 +438,7 @@ public function loadNode($element, $name = null)
         $this->setFrete(Util::loadNode($element, 'vFrete'));
         $this->setDespesas(Util::loadNode($element, 'vOutro'));
         $this->setTributos(Util::loadNode($element, 'vTotTrib'));
+        $this->setTotalNota(Util::loadNode($element, 'vNF'));
```

Comment:
o nome da tag é v de Valor e NF = Nota Fiscal, o nome do arquivo NFe (Nota Fiscal Eletrônica) Total (Total), logo  o nome do campo deveria ser apenas Valor, logo `$this->setValor(...)`